cloud/cloud.go

```diff
@@ -96,6 +103,13 @@ type (
 	}
 )
 
+// Available cloud locations.
+const (
+	EU Region = iota
```

Comment:
IMHO it's strange to use `invalidRegion` as latest value in const enum instead of the first one and keep default EU as `iota+1`

Reply:
because the `cloud.Client` has no constructor function, the idea is that just doing `cloud.Client{}` will automatically configure `cloud.Client.Region` as `EU` by default (using the zero value). So it uses the zero value.

I actually forgot to remove the `invalidRegion` because before I was "validating" the region with `if c.Region >= invalidRegion { return errors.E(...) }` but later decided to just validate for 'EU` and `US` for now, as I don't think this enum will be extended further ever.

Reply:
actually I will need `invalidRegion` in the last PR

Reply:
I noticed in other PRs I'd say to add a comment because it's really confusing

Reply:
I will add a comment but just to clarify further: we usually make the first enum the illegal one to avoid the zero value misrepresenting something. But here we actually do want to make EU the default region.

but yes, a comment makes sense because usually you want to prevent that.
In the future, if we introduce a construtor to the `cloud.Client` then I think makes sense to make the zero value illegal.

Thank you!